app/boards/arm/bluemicro840/bluemicro840_v1.dts

```diff
@@ -29,6 +29,7 @@
 	ext-power {
 		compatible = "zmk,ext-power-generic";
 		label = "EXT_POWER";
+		init-delay-ms = <12>;
```

Comment:
Just to be safe, can we bump to 15 or 20?

Reply:
Sure, bumped and rebased from main